### PR TITLE
Patch footer

### PR DESCRIPTION
**Pull request by jsbUSMC**
React threw a warning when rendering the Footer without using a key prop. The current Footer class contains the function renderGitHubFollowButton() and maps the AUTHORS const returning the two buttons for the authors of the library. I have changed the const to contain an id field, and use this id as the key in a div that wraps the old function's returned jsx. I have changed the contents of the template strings to reflect "author.name" in the relevant places as well.

### CODE REVIEW

File: src/components/Footer/index.js [hunk 1]
```diff
@@ -4,21 +4,18 @@ import React, { Component } from 'react';
...
-const AUTHORS = ['anorudes', 'keske'];
+const AUTHORS = [
+  {
+    id: 1,
+    name: 'anorudes'
+  }, {
+    id: 2,
+    name: 'keske'
+  }
+];
```
Comment (jsbUSMC) [inline src/components/Footer/index.js:15]: added 'id' field for the key

File: src/components/Footer/index.js [hunk 2]
```diff
@@ -4,21 +4,18 @@ import React, { Component } from 'react';
...
-      return (
-        <a className="github-button" href={`https://github.com/${user}`}
-           data-count-href={`/${user}/followers`}
-           data-count-api={`/users/${user}#followers`}
-           data-count-aria-label="# followers on GitHub"
-           aria-label={`Follow @${user} on GitHub`}>
-          @{user}
-        </a>
-      );
-    }
```
Comment (jsbUSMC) [inline src/components/Footer/index.js:21]: This function no longer necessary, but used all the jsx in the new map function below, changing only the string template contents.

**Comment by anorudes** [comment]
Thanks bro :+1:

**Comment by keske** [comment]
Better way: https://github.com/anorudes/redux-easy-boilerplate/commit/3303828439ac640faefa8d51e5111239783e4c21

**Pull request merged**